features/dd-sdk-android-logs/src/main/json/log/log-schema.json

```diff
@@ -6,10 +6,10 @@
   "description": "Structure holding information about a Log",
   "properties": {
     "device": {
-      "$ref": "_common-schema.json#/properties/device"
+      "$ref": "../../../../../dd-sdk-android-rum/src/main/json/rum/_common-schema.json#/properties/device"
     },
     "os": {
-      "$ref": "_common-schema.json#/properties/os"
+      "$ref": "../../../../../dd-sdk-android-rum/src/main/json/rum/_common-schema.json#/properties/os"
```

Comment:
Such approach does not keep the file which was used to generate code. In case if any trouble we can't find source for the generated code. Plus, if you make any changes in the `dd-sdk-android-rum` project (even if you change the branch) it will lead to files re-generation. What is the advantage we gonna have with such configuration compared to the previous one?